src/axom/sina/core/Document.hpp

```diff
@@ -284,6 +298,78 @@ Document loadDocument(std::string const &path,
                       RecordLoader const &recordLoader,
                       Protocol protocol = Protocol::JSON);
 
+/**
+ * \brief Append the new records or, per-record, new data, user defined content, curves/curve sets,
+ *        and library data of a Sina Document to an existing JSON file. For a full explanation of behavior,
+ *        see append_to_hdf5. Note that a JSON file must be entirely re-written for an append to function;
```

Comment:
Minor: Should this be `appendDocumentToHDF5` instead of `append_to_hdf5`?

```suggestion
 *        see appendDocumentToHDF5. Note that a JSON file must be entirely re-written for an append to function;
```